packages/sessions-sdk-react/src/components/modal-dialog.module.scss

```diff
@@ -5,25 +5,27 @@
     position: fixed;
     top: 0;
     left: 0;
-    width: 100vw;
-    height: var(--visual-viewport-height);
+    height: 100dvh;
+    width: 100dvw;
     background: theme.color("modal-overlay");
     backdrop-filter: blur(8px);
     display: grid;
     justify-content: center;
     align-items: start;
-    padding-top: theme.spacing(40);
     z-index: theme.layer("modal-dialog");
+    overflow: auto;
```

Comment:
recommend also putting `-webkit-overflow-scrolling: touch` to ensure smooth scrolling on mobile devices (the inertial scrolling behavior you get in native apps)